app/views/application/_footer.haml

```diff
@@ -11,6 +11,9 @@
       %li
         Read our blog
         = link_to "openaustraliafoundation.org.au/blog", "http://www.openaustraliafoundation.org.au/blog"
+      %li
+        Fork this project
```

Comment:
"Fork" is a bit jargony for this project - the general audience aren't programmers so wouldn't understand what it meant. Can we improve this wording?